packages/mirrorful/editor/src/components/Dashboard.tsx

```diff
@@ -44,6 +44,18 @@ export function Dashboard() {
     onExportSuccessModalOpen()
   }
 
+  const handleUpdate = async (data: TColorData[]) => {
+    setColors(() => {
+      return data
+    })
```

Comment:
I think this can be simplified as `setColors(data)`. 

```suggestion
    setColors(data)
```

Reply:
Of course - (testing remnants while I'm finding my feet in new codebase).  Apologies!

Reply:
All good!

Reply:
Thanks for the help @alexdanilowicz !